City-of-Pori/pori-kada
# PORI-577: Contact card styles

Adds styling to contact cards as per designs. Added new dimensions for person related image styles to achieve square image.

Steps to test:
1. See that the contact person section looks good and is according to designs, compare these:
[news page in dev](https://businesspori.dev.wunder.io/uutinen/2019-11-05_uutisen-esimerkki-4
) and [news page in designs](https://xd.adobe.com/spec/71fab5ee-33a5-4fb9-7201-fbf8e406e23e-07cb/screen/54bdab9d-0b23-4f10-9415-71464478938f/1-2-businesspori-fi-Porin-edut-Mahdollistava-infra)

2. See that [contact persons list in dev](https://businesspori.dev.wunder.io/test-page-bp-contact-card) can acheive same mockup as in [designs](https://xd.adobe.com/spec/71fab5ee-33a5-4fb9-7201-fbf8e406e23e-07cb/screen/53cfc947-0452-45ff-842e-64ad45e9080a/4-businesspori-fi-ota-yhteytt-):  

3. See that transparent images are supported in contact card image. See that pori.fi contact cards were mostly unaffected (new image sizes also affected other domains but otherwise they should look as before)

4. Check that mobile looks good too.
